### PR TITLE
Add Ferre-2022-160

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

Based on issue #1165.

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
@LinguList @xrotwang I had an error while adding the concept list. I'm guessing that it is due to the datatypes for the columns ORTHOGRAPHIC_SIMILARITY or one of the "EMOTION" columns. But I couldn't figure out what caused the problem. Could you check the metadata.json file?

**Comment by LinguList** [comment]
The error code seems to point to a generalnüroblem of your json metadata file. Maybe a comma missing? 
```
json.decoder.JSONDecodeError: Expecting value: line 107 column 17 (char 3999)
========
```

It is in row 17 or before or after it.

**Comment by LinguList** [comment]
A comma in the end of a list/dictionary. Line 106. Do you see it? The list starts in line 17, that is why it stops there...

**Comment by AnnikaTjuka** [comment]
> A comma in the end of a list/dictionary. Line 106. Do you see it? The list starts in line 17, that is why it stops there...

Thanks! I was staring at the file for so long that I didn't see it. Now all checks work.

**Comment by AnnikaTjuka** [comment]
@martino-vic Do you have time to review the PR? The list only includes nouns and adjectives.

File: concepticondata/conceptlists/Ferre-2022-160.tsv [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+Ferre-2022-160-28	28			melancholy	melancólico	3.19	1.6	1.76	1.29	5.63	4.81	6.6	4.23	1	2	1	0.55
+Ferre-2022-160-29	29	56	BAD LUCK	misfortune	infortúnio	0.87	0.45	0.3	0.35	8.26	6.18	7.6	3.84	1	2	1	0.5
+Ferre-2022-160-30	30			nervousness	nervosismo	1.87	1.22	1.88	0.71	6.9	5.75	5.95	5.8	1	2	1	0.55
+Ferre-2022-160-31	31	134	FAULT	guilt	culpa	1	0.65	1.1	0.14	8.73	7.04	7.95	6.46	1	2	1	0.2
+Ferre-2022-160-32	32			humiliation	humilhação	0.69	0.53	0.1	0.06	8.74	7.31	9.05	7.49	1	2	1	0.64
+Ferre-2022-160-33	33			despair	desespero	0.76	0.58	0.98	0.17	8.69	6.36	7.29	4.13	1	2	1	0.44
+Ferre-2022-160-34	34	1000	ANXIETY	anxiety	ansiedade	0.68	0.64	1.38	0.17	8.73	7.05	8.05	7.74	1	2	1	0.44
+Ferre-2022-160-35	35	40	ENVY	envy	inveja	1.13	0.91	1.15	0.29	7.89	6	7.6	5.62	1	2	1	0.33
+Ferre-2022-160-36	36	1141	PITY	pity	pena	1.38	1.24	1.36	0.52	7.51	6.22	6.08	3.94	1	2	1	0.25
+Ferre-2022-160-37	37	1783	PAIN	agony	agonia	0.61	0.76	0.2	0.15	8.73	6.76	8.71	6	1	2	1	0.67
```
Comment (Tarotis) [inline concepticondata/conceptlists/Ferre-2022-160.tsv:38]: I would unmap, as `PAIN` is much more general than "agony"

File: concepticondata/references/references.bib [hunk 2]
```diff
@@ -1,3 +1,13 @@
+@article{Ferre2022,
+  title = {Love me in {{L1}}, but hate me in {{L2}}: {{How}} native speakers and bilinguals rate the affectivity of words when feeling or thinking about them},
+  author = {Ferr{\'e}, Pilar and Guasch, Marc and {Stadthagen-Gonzalez}, Hans and Comesa{\~n}a, Montserrat},
+  year = {2022},
+  volume = {3},
+  journal = {Bilingualism: Language and Cognition},
+  pages = {1-15},
```
Comment (Tarotis) [inline concepticondata/references/references.bib:7]: I get the following information for volume and pages from the journal website:

| Bilingualism: Language and Cognition, 25(5), 786-800

So this should be adapted for number, issue, and pages

**Review by Tarotis — CHANGES_REQUESTED** [review]
I have only two small comments that refer to a) probably unmapping one concept, and b) adapting the bib-entry. Other than that, this PR looks ready to go.

File: concepticondata/references/references.bib (hunk 2 above)
Comment (AnnikaTjuka) [reply concepticondata/references/references.bib:7]: Thanks! I downloaded the PDF a while ago so they must have updated it.

**Review by Tarotis — APPROVED** [review]
Thanks for the update, from my side everything looks ready to merge now.

**Comment by AnnikaTjuka** [comment]
Great! I'll merge now.

**Pull request merged**